enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,88 @@
-const enhancer = require('./enhancer.js');
-// test away!
+const enhancer = require("./enhancer.js");
+
+const item1 = {
+  name: "Iron Sword",
+  enhancement: 12,
+  durability: 50
+};
+
+const item2 = {
+  name: "Golden Scythe",
+  enhancement: 4,
+  durability: 70
+};
+
+const item3 = {
+  name: "Excalibur",
+  enhancement: 20,
+  durability: 100
+};
+
+test("Should return an item with the durability restored to 100.", function() {
+  expect(enhancer.repair(item1)).toEqual({
+    name: item1.name,
+    enhancement: item1.enhancement,
+    durability: 100
+  });
+  expect(enhancer.repair(item2)).toEqual({
+    name: item2.name,
+    enhancement: item2.enhancement,
+    durability: 100
+  });
+  expect(enhancer.repair(item3)).toEqual({
+    name: item3.name,
+    enhancement: item3.enhancement,
+    durability: 100
+  });
+});
+
+test("Should return an item with the enhancementt 1 point higher, unless it is 20", function() {
+  expect(enhancer.succeed(item1)).toEqual({
+    name: item1.name,
+    enhancement: 13,
+    durability: item1.durability
+  });
+  expect(enhancer.succeed(item2)).toEqual({
+    name: item2.name,
+    enhancement: 5,
+    durability: item2.durability
+  });
+  expect(enhancer.succeed(item3)).toEqual({
+    name: item3.name,
+    enhancement: 20,
+    durability: item3.durability
+  });
+});
+
+test("Should return an item with durability decreased by 5 if enhancement is lower than 15, else it decreases by 10. Should return enhancement decreased by 1 if it was greater than 16", function() {
```

Comment:
having all these wrapped in a single `test` block is good, but make it more readable in the string description: _what_ should return these values? `fail()`. take a look at how Luis did his demo, it's a very good example of superbly readable test blocks: one `describe` block describes the whole function, then it has multiple `it` tests within that declare the specific case being tested